frontend/packages/kubevirt-plugin/console-extensions.json

```diff
@@ -12,7 +12,7 @@
       "model": {
         "group": "tektontasks.kubevirt.io",
         "version": "v1alpha1",
-        "kind": "TektonTasks"
+        "kind": "NotExists"
```

Comment:
NOOOOO, this is a debugging comment !!!!
should not merge this
@pcbailey @rawagner 

/hold

Reply:
Fixed!